### DISCUSSION TITLE
Next step towards this project contribution

### DISCUSSION THREAD

**Discussion by coolsandeee** [scaleracademy/twitter-backend-java#23] [General]
I have set up this project in my local and able to post/get requests as well.
What is my next step towards this project contribution? 
Kindly guide me.

**Comment by ohbus** [comment]
There are few APIs (mostly the unauthenticated ones) that we have created.

I have lately been occupied with some office work and have not been able to complete the authenticated APIs.

You may clone [this branch](https://github.com/ohbus/twitter-backend/tree/testing) and continue building the code where we have left from this #18 PR

Though we have not been keeping a track of it as of now, it is always a good idea to start doing it.
